news/_posts/translation/do-you-speak-prestashop-nov-2020.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Do you speak PrestaShop? – November 2020 edition"
+subtitle: "Where you'll learn that the Hebrew project is progressing"
+date:  2020-12-08 17:00:00
+authors: [JulieVarisellaz]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it is also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in November!
+
+[Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+
+### The Hebrew project is progressing!
+
+In just a month, the Hebrew project went from 64% to 83% translated and validated, an increase of 19%! What a great evolution! :scream: Congratulations! Special thanks to  [danielshapiro9](https://crowdin.com/profile/danielshapiro9) who has joined us just about a month ago and who has already translated 26,091 words. Thank you for your involvement! And of course, thank you to the other translators of the Hebrew project!
```

Comment:
Hey, the first sentence needs to be updated as the graph in ## Languages with the best evolution mentions an increase of 20%. :-)

Reply:
```suggestion
In just a month, the Hebrew project went from 64% to 83% translated and validated, an increase of 19%! What a great evolution! :scream: Congratulations! Special thanks to [danielshapiro9](https://crowdin.com/profile/danielshapiro9) who has joined us just about a month ago and who has already translated 26,091 words. Thank you for your involvement! And of course, thank you to the other translators of the Hebrew project!
```